clueless/models.py

```diff
@@ -226,6 +226,7 @@ class Game(models.Model):
     startTime = models.DateTimeField(default = datetime.datetime.now, blank = True)
     lastUpdateTime = models.DateTimeField(default=datetime.datetime.now, blank=True)
     hostPlayer = models.ForeignKey(Player)
+    name = models.CharField(max_length=60)
 
```

Comment:
Ah yeah I saw that as an option but I feel like unless it's problematic to the functionality of the game specifying the default in the code might be good so that someone running make migrations doesn't have to specify something.

nuking the database sort of worked except that it didn't actually fully nuke the database (see other comment)